go/plumbing/operations/upload_deploy_function_parameters.go

```diff
@@ -62,6 +62,8 @@ for the upload deploy function operation typically these are written to a http.R
 */
 type UploadDeployFunctionParams struct {
 
+	/*XNfRetryCount*/
+	XNfRetryCount *int64
```

Comment:
nit: do you really need that big number or something like `uint8` could wokr?

Reply:
This code is automatically generated by Swagger. It seems to translate `type: integer` into `int64`, and I'm not sure we can configure that.

Reply:
On 64bit system the integer size doesn't make any difference